apps/cow-fi/app/(main)/cow-amm/page.tsx

```diff
@@ -73,11 +73,11 @@ export default function Page() {
           <p>more TVL achieved than reference pool (beta phase)</p>
         </MetricsItem>
         <MetricsItem dividerColor="#9BD955">
-          <h2>$11M+</h2>
+          <h2>$18M+</h2>
           <p>liquidity protected from LVR</p>
         </MetricsItem>
         <MetricsItem>
-          <h2>$90K+</h2>
+          <h2>$1.2M+</h2>
           <p>surplus captured for LPs (beta phase)</p>
```

Comment:
Is this data still in the beta phase?

Reply:
IDK. No one confirmed [this ](https://cowservices.slack.com/archives/C067ZLV8U13/p1739551989849209) on my request to help with numbers. I received a  controversial feature-request here. So i decided to change a number and that it.